sources/packages/api/src/services/application/application.service.ts

```diff
@@ -15,11 +15,15 @@ import {
   COEStatus,
   ProgramYear,
 } from "../../database/entities";
-import { SequenceControlService } from "../../services/sequence-control/sequence-control.service";
-import { CreateApplicationDto } from "../../route-controllers/application/models/application.model";
 import { CustomNamedError } from "../../utilities";
+import { SequenceControlService } from "../sequence-control/sequence-control.service";
+import { StudentFileService } from "../student-file/student-file.service";
 
 export const PIR_REQUEST_NOT_FOUND_ERROR = "PIR_REQUEST_NOT_FOUND_ERROR";
+export const APPLICATION_DRAFT_NOT_FOUND = "APPLICATION_DRAFT_NOT_FOUND";
+export const ONLY_ONE_DRAFT_ERROR =
+  "ONLY_ONE_APPLICATION_DRAFT_PER_STUDENT_ALLOWED";
```

Comment:
Can we please rename it to MORE_THAN_ONE_APPLICATION_DRAFT_ERROR or SINGULAR_APPLICATION_DRAFT? I feel, ONLY_ONE_DRAFT_ERROR does not get me the understanding of what it is

Reply:
Done